step-templates/wait.json

```diff
@@ -1,16 +1,20 @@
 {
-  "Id": "c69b03c9-bfa1-41ab-a712-ba74dc0512b1",
-  "Name": "Wait",
+  "Id": "03ef1bf5-4ca1-4df0-a013-6b49d9c565f6",
```

Comment:
`Id` should not be updated as this will break the update integration with Octopus Server.